docs/options.md

```diff
@@ -172,6 +172,8 @@ Options:
 - macOS: `x86_64` `arm64` `universal2`
 - Windows: `AMD64` `x86`
 - `auto`: The default archs for your machine - see the table below.
+    - `auto64`: Just the 64-bit auto archs (will result in an empty set on a 32-bit runner)
+    - `auto32`: Just the 32-bit auto archs (will result in an empty set on macOS)
```

Comment:
Maybe we could remove the parens here, but add auto64 and auto32 columns to the table below?